### DISCUSSION TITLE
Impossible to override checkbox focus styling

### DISCUSSION THREAD

**Discussion by DenisCor** [carbon-design-system/carbon#13290] [Help] [answered]
```
.cds--checkbox:focus {
  outline: none !important;
}
```

This wont work!

This is for TableSelectRow and TableSelectAll components.

EDIT: Fixed.

had to also include 
```
.cds--checkbox - label {
    & : before {
        outline: none;
    }
}
```
which is weird..

**Comment by tw15egan** [comment, marked as answer]
We do not recommend overriding focus styles, as this could potentially introduce a11y violations
